include/oneapi/dpl/pstl/hetero/dpcpp/sycl_traits.h

```diff
@@ -176,6 +224,9 @@ struct __partition_by_mask;
 template <typename _Inclusive, typename _BinaryOp, typename _InitType>
 struct __global_scan_functor;
 
+template <typename _InitType>
+ struct __init_value;
```

Comment:
Is it just a forward declaration of some struct we should specialize for?

Reply:
Yes.  It was a type that was missed in the first round.  It was made clear in the test for `__scan` that this was necessary.  It is specialized for device copyable below.